src/pages/products/product-add/product-add-submit/product-add-submit.js

```diff
@@ -95,7 +95,9 @@ const ProductAddSubmit = ({
           ? productToSend[name].find((product) => product.lang === lang).value
           : productToSend[name];
 
-        return text ? <Detail key={name} title={label} text={text} /> : null;
+        return text ? (
```

Comment:
We have lodash package for it in our project. https://lodash.com/docs/4.17.15#map